ci/ray_ci/tester_container.py

```diff
@@ -140,6 +142,21 @@ def upload_test_results(cls, team: str, bazel_log_dir: str) -> None:
             test.persist_to_s3()
             test.persist_test_result_to_s3(result)
 
+    @classmethod
+    def move_test_state(cls, team: str, bazel_log_dir: str) -> None:
+        pipeline_id = os.environ.get("BUILDKITE_PIPELINE_ID")
```

Comment:
maybe check `RAYCI_CONTINUOUS_BUILD == 1`?

I also worry that if somehow there is a bug in the test where prevents the test to reach this step (e.g. oom that kills the machine), then the state machine will be just silently stop moving.

Reply:
i intentionally run this every time we have new test results instead of on continuous build (so that we can open and close tickets quicker)

> a bug in the test where prevents the test to reach this step 

we can build metric + monitoring system across all of our tools yes; for this case the metric is the staleness of data (they are updated on every test run presumably), an extra protection on top of test cases